common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
I like the idea of having two classes: `is-recent-contributor` and `is-member`, because it allows to react independently on both scenarios. They are easy to read and memorable.

What do you think? I'd leave out any responsive additions for now (this could also be handled in inlined media queries) :)